index.html

```diff
@@ -2624,12 +2618,17 @@ <h2>WoT Runtime</h2>
             <p>
                 Technically, the <a>Thing</a> abstraction and its <a>Interaction Model</a> is implemented in a runtime system.
                 This <a>WoT Runtime</a> maintains the execution environment for the behavior implementation and is able to expose and/or consume <a>Things</a>,
-                and hence must be able to process, serialize, serve, and fetch <a>WoT Thing Descriptions</a>.
+                and hence must be able to fetch, process, serialize, and serve <a>WoT Thing Descriptions</a>.
             </p>
             <p>
-                The <a>WoT Runtime</a> must provide an API for the behavior implementation that follows the <a>Thing</a> abstraction.
-                The optional <a>WoT Scripting API</a> building block shown in <a href="#architecture-implementation"></a> defines such an application-facing interface.
+                The <a>WoT Runtime</a> must provide an API for the behavior implementation.
+                The optional <a>WoT Scripting API</a> building block shown in <a href="#architecture-implementation"></a>
+                defines such an application-facing interface that follows the <a>Thing</a> abstraction.
                 See <a href="#native-impl"></a> for alternative APIs.
+                In general, application logic should be executed in isolated <a>Execution Environments</a>
+                that prevent unauthorized access to the management aspects of the <a>WoT Runtime</a>,
+                in particular the private security configurations.
+                In multi-tenant <a>Servients</a>, additional <a>Execution Environment</a> isolation is required for the different tenants.
```

Comment:
an execution environment might be shared, it is the execution contexts that are isolated

Reply:
Won't fix (see above)

Reply:
Should fix (see above).